news/1472.api_change

```diff
@@ -0,0 +1 @@
+Optuna sweeper config stucture changes, please refer to the [plugin docs](https://hydra.cc/docs/next/plugins/optuna_sweeper/)
```

Comment:
there should be a news directory inside the Optuna sweeper dir (if there isn't make one).
move your two news fragments into it.